pkg/generate/command.go

```diff
@@ -0,0 +1,35 @@
+package generate
+
+import (
+	"fmt"
+	"log"
+	"github.com/budimanjojo/talhelper/pkg/config"
+	"github.com/budimanjojo/talhelper/pkg/talos"
+)
+
+func GenerateCommand(c *config.TalhelperConfig, outDir string, generateForNode string, generateApply bool, generateUpgrade bool) error {
+	if !generateApply && !generateUpgrade {
+		fmt.Printf("Must select one of `--apply` of `--upgrade`\n")
+		return nil
+	}
+
+	for _, node := range c.Nodes {
+		var isSelectedNode = generateForNode != "" && generateForNode == node.IPAddress
+		if generateForNode == "" || isSelectedNode {
+			if generateApply {
+				fileName := outDir + "/" + c.ClusterName + "-" + node.Hostname + ".yaml"
+				fmt.Printf("talosctl apply-config --talosconfig %s/talosconfig --nodes %s --file %s --insecure;\n", outDir, node.IPAddress, fileName)
+			}
+			
+			if generateUpgrade {
+				var image, err = talos.GetInstallerURL(node.Schematic, "factory.talos.dev", c.TalosVersion)
+				if err != nil {
+					log.Fatalf("Failed to generate installer url, %v", err)
```

Comment:
return the error instead `return fmt.Errorf("failed to generate installer url: %v", err)`

There are more same cases below